### PR TITLE
added volume indicator

### PR DESCRIPTION
**Pull request by ankeshni**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: Project/src/MakeCall/CallCard.js [hunk 1]
```diff
@@ -111,12 +117,36 @@ export default class CallCard extends React.Component {
                 });
             });
 
+            let localVolumeStateSetter = undefined;
+            let handleSelectedMichrophoneVolumeSubscription = async () => {        
+                let localVolumeIndicator = await (new LocalAudioStream(this.deviceManager.selectedMicrophone).getVolume());
+                localVolumeStateSetter = ()=>{
+                    this.setState({ localVolumeLevel: localVolumeIndicator.level });
+                }
+                localVolumeIndicator.on('levelChanged', localVolumeStateSetter);
```
Comment (akania) [inline Project/src/MakeCall/CallCard.js:126]: this may be very chatty and cause lots of UI updates, please test it and if needed , debounce UI updates

Comment (ankeshni) [reply Project/src/MakeCall/CallCard.js:126]: The volume indicator bar transitions are smooth (sampled every 100ms), and things like smoothning constant and sampling time are ECS configurable.

Comment (fizampou) [reply Project/src/MakeCall/CallCard.js:126]: y lets debounce to every min or two :)

**Review by pkestikar — APPROVED** [review]

File: Project/src/MakeCall/CallCard.js [hunk 2]
```diff
@@ -111,12 +117,36 @@ export default class CallCard extends React.Component {
...
+                this.setState({ localVolumeLevelSubscription: localVolumeStateSetter });
+                this.setState({ localVolumeIndicator: localVolumeIndicator });                             
+            }
+            handleSelectedMichrophoneVolumeSubscription();
```
[the remark this answers is not in the document]
Comment (ankeshni) [reply Project/src/MakeCall/CallCard.js]: fixed

**Review by csandjon — APPROVED** [review]

**Review by fizampou — APPROVED** [review]

**Pull request merged**